datalad/downloaders/tests/test_s3.py

```diff
@@ -158,3 +158,12 @@ def test_boto_host_specification(tempfile):
     with swallow_outputs():
         providers.download(url_dandi1, path=tempfile)
     assert_equal(md5sum(tempfile), '97f4290b2d369816c052607923e372d4')
+
+
+def test_restricted_bucket_on_NDA():
```

Comment:
So, this only runs on your local machine, correct?

Reply:
yes unfortunately.  NDA access, and to this specific dataset is required and it is a not that easy procedure to get. And with their specific settings for the bucket - the value from the test is really in talking to NDA.
May be at some point we could setup a github workflow worker which would sit on "trusted" infrastructure and has access to the credentials, but then we might need to (as I think I did with buildbot in the past) to make it run tests only if PRs submitted by trusted github users or a dedicated label is added to signal that PR diff is good (no malicious code which would expose secrets) to be tested on